release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2643,3 +2643,17 @@ link:https://access.redhat.com/solutions/6324771[{product-title} 4.8.11 containe
 ==== Upgrading
```

Comment:
Also, worth adding something to [the update docs][1]?  I'm actually a bit fuzzy on whether 4.8's minor-bump docs are about 4.7 -> 4.8 (makes sense based on `stable-4.8` and similar channels being discussed), or about 4.8 -> 4.9 (makes sense based on the 4.8 oc/console UX being discussed).

[1]: https://docs.openshift.com/container-platform/4.8/updating/updating-cluster-between-minor.html

Reply:
@wking The upgrading docs are about upgrading to the current version. So the 4.8 upgrading docs are about upgrading tfrom 4.7 -> 4.8. 

So no, we'd be mixing versions if we added something talking about 4.9 there. I plan to add this information (from the kbase) to the Upgrading section of the 4.9 docs.

Reply:
So... [here][1] we picked docs about `oc adm upgrade channel ...` back into 4.9's minor-update section.  If that section is about using the 4.8 tooling to get to 4.9, perhaps we should revert that change?  While it is possible to use a 4.9 `oc` to ask a 4.8 cluster to move to 4.9, I would expect folks using `oc` to talk to 4.8 clusters to imply 4.8 `oc` unless we explicitly call out another version.  And folks talking to 4.8 clusters will definitely be using 4.8 web consoles.  Anyhow, I'm happy to move this thread elsewhere, since "4.8's minor-updates section is about 4.7 -> 4.8" is enough to know we don't want any of this admin-ack content over there.

[1]: https://github.com/openshift/openshift-docs/pull/34900/files#diff-0d1d62e12d72baebd033e4bc283a1e6d3e6c3a8dc16ed97bc461dc1e7386fffbR12

Reply:
Agree. If "4.8's minor-updates section is about 4.7 -> 4.8", we might need to place some similar instructions in 4.9 release notes, and also deserve a dedicated section to guide administrator acknowledgment in 4.9 official doc.
For 4.8.z release notes, might we just need some short description to notify folks that this z-stream release introduces `admin-gates` configmap about "ack-4.8-kube-1.22-api-removals-in-4.9" in `openshift-config-managed` namespace. Of course, the present descriptions in this PR for 4.8.z release notes, looks good to me.